browser/brave_content_browser_client.cc

```diff
@@ -224,6 +224,7 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #if !BUILDFLAG(IS_ANDROID)
 #include "brave/browser/new_tab/new_tab_shows_navigation_throttle.h"
 #include "brave/browser/ui/geolocation/brave_geolocation_permission_tab_helper.h"
+#include "brave/browser/ui/webui/brave_education/brave_education_ui.h"
```

Comment:
why is this under `!IS_ANDROID`?

Reply:
Updated to use a new buildflag `enable_brave_education`.